assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
```

Comment:
Here is how it looks without the breakpoint at 977px. The banner should be centred, instead it is taking the full width.

![beta_banner_at_986px](https://cloud.githubusercontent.com/assets/15999736/13351735/c29469c8-dc80-11e5-935a-dfa9a8ab9cb3.png)